src/signals/incident/definitions/wizard-step-1-beschrijf.js

```diff
@@ -83,25 +84,14 @@ export default {
       subcategory: {
         meta: {
           label: 'Subcategorie',
+          path: 'subcategory',
           type: 'text',
         },
         options: {
           validators: [Validators.required],
         },
         render: FormComponents.HiddenInput,
       },
-      subcategory_link: {
-        meta: {
-          label: 'Subcategorie',
-          path: 'category.sub_category',
-          type: 'text',
-          validateAsyncProp: 'incidentContainer.loadingClassification',
-        },
```

Comment:
die validateAsyncProp zorgt ervoor dat er wordt gewacht met het ophalen van de  classificatie na het drukken op de submit knop. dat werkt niet meer in jouw branch

Reply:
Seems to go smoothly; Redux actions are executed in order and the `incident` scope is updated with the proper category and subcategory on blur or submit:

<img width="1439" alt="Screenshot 2020-02-11 at 16 36 49" src="https://user-images.githubusercontent.com/1062191/74251639-b9b71e00-4cec-11ea-9b8f-f6480dd6b713.png">

Reply:
And I took out the `async` behaviour, because it wasn't needed anymore; not retrieving the full list of categories.